cmd/ui/package.json

```diff
@@ -64,6 +64,7 @@
     },
     "devDependencies": {
         "@faker-js/faker": "^9.0.3",
+        "@rollup/pluginutils": "^5.3.0",
```

Comment:
Doodle was using a newer version of a Vite plugin which provides HMR which conflicted with the 4.2.1 version. Because of the `^`, the package manager was resolving to the later version 4.7.0 which  depends are react-fresh 0.17.0 which introduces a breaking change. Explicitly setting the older version prevents this.